### PR TITLE
Update Centurion banlist and lifepoints

### PR DESCRIPTION
**Pull request by andreacosta**

### CODE REVIEW

**Comment by andreacosta** [comment]
@theelk801 hi, i would just update banlist and lifepoints. What happen with Travis? i don't know what is exactly

**Comment by JayDi85** [comment]
You can see full errors log by click on travis details:
shot_200902_135334

Error:
> /home/travis/build/magefree/mage/Mage.Server.Plugins/Mage.Deck.Constructed/src/mage/deck/CenturionCommander.java:[68,38] ';' expected

File: Mage.Server.Plugins/Mage.Deck.Constructed/src/mage/deck/CenturionCommander.java [hunk 1]
```diff
@@ -66,21 +64,24 @@ public CenturionCommander() {
         banned.add("Tolarian Academy");
         banned.add("Treasure Cruise");
         banned.add("Vampiric Tutor");
-        banned.add("Vanishing");
-        banned.add("Winter Orb");
         banned.add("Yawgmoth's Bargain");
+        banned.add("Thassa's Oracle")
```
Comment (JayDi85) [inline Mage.Server.Plugins/Mage.Deck.Constructed/src/mage/deck/CenturionCommander.java:68]: missing `;`

Comment (andreacosta) [reply Mage.Server.Plugins/Mage.Deck.Constructed/src/mage/deck/CenturionCommander.java:68]: oh shit

Comment (andreacosta) [reply Mage.Server.Plugins/Mage.Deck.Constructed/src/mage/deck/CenturionCommander.java]: @JayDi85 thanks for your help, i not use this a lot

**Pull request merged**